volatility3/framework/plugins/linux/malfind.py

```diff
@@ -88,6 +85,7 @@ def _generator(self, tasks):
                         process_name,
                         format_hints.Hex(vma.vm_start),
                         format_hints.Hex(vma.vm_end),
+                        vma_name or renderers.NotAvailableValue(),
```

Comment:
Should be ok, since the generator is private and the TreeView that eventually gets returned from `Run` should be inspectable/self-describing, so I think this is fine.